core/iwasm/common/wasm_runtime_common.c

```diff
@@ -3597,6 +3597,103 @@ static union {
 
 #define is_little_endian() (__ue.b == 1) /* NOLINT */
 
+uint32_t
+wasm_runtime_import_count(const wasm_module_t module)
+{
+    if (!module) {
+        return 0;
```

Comment:
refer to https://github.com/bytecodealliance/wasm-micro-runtime/pull/3330/files#r1569850911